src/default.vert.glsl

```diff
@@ -5,14 +5,14 @@
 uniform mat4 camera;
 
 attribute vec3 position;
-attribute int textureid;
+attribute float textureid;
```

Comment:
I see you made this change on the GLSL side of things, but I'm not seeing any changes in this commit telling C about this change. Might cause some issues later if we aren't careful.